rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -403,25 +411,42 @@ def _deserialize_channel_withdraw(
     ) -> ChannelWithdraw:
         """Deserialize a channel withdraw event. Only for Tom pool.
 
-        It may raise ChannelTokenAddressError, KeyError and ValueError.
+        May raise DeserializationError.
         """
         inverse_identity_address_map = {
             address: identity for identity, address in identity_address_map.items()
         }
-        tx_hash, _, log_index = raw_event['id'].split(':')
-        address = to_checksum_address(raw_event['user'])
-        identity_address = inverse_identity_address_map[address]
-        amount = FVal(raw_event['amount']) / ADX_AMOUNT_MANTISSA
-        channel_id = raw_event['channel']['channelId']
-        token_address = to_checksum_address(raw_event['channel']['tokenAddr'])
+        try:
+            tx_hash, _, log_index = raw_event['id'].split(':')
+            address = to_checksum_address(raw_event['user'])
+            identity_address = inverse_identity_address_map[address]
+            amount = FVal(raw_event['amount']) / ADX_AMOUNT_MANTISSA
+            channel_id = raw_event['channel']['channelId']
+            token_address = to_checksum_address(raw_event['channel']['tokenAddr'])
+        except (KeyError, ValueError) as e:
+            msg = (
+                'Failed to deserialize an AdEx channel withdraw event. '
+                'Check logs for more details'
+            )
+            log.error(
+                msg,
+                error=str(e),
```

Comment:
This is not good. str(e) for both a KeyError and a generic ValueError are not sufficient. For a `KeyError` str(e) is just the missing key and for a `ValueError` it depends, but in your case it's gonna be something like `"not enough values to unpack (expected 2, got 1)"`

As I said below please:

Handle ValueError and reraise it as DeserializationError mentioning the attribute for which the problem was hit.

For KeyError you can get away with something like what we do [here](https://github.com/rotki/rotki/blob/70508f99f890bcbd520f1efe7776194d6a5e5e06/rotkehlchen/exchanges/poloniex.py#L185-L198)  poloniex as I said in another comment.